src/main/java/org/opensearch/jdbc/types/BaseTypeConverter.java

```diff
@@ -35,6 +36,9 @@ public abstract class BaseTypeConverter implements TypeConverter {
         typeHandlerMap.put(Date.class, DateType.INSTANCE);
         typeHandlerMap.put(Time.class, TimeType.INSTANCE);
 
+        // Adding Struct Support
```

Comment:
np: the comment seems not very necessary. suggest to remove it. Same for the one in OpenSearchType.

Reply:
Removed.